components/settings/Account/Billing/Alert.tsx

```diff
@@ -33,7 +37,7 @@ type DismissedStorage = {
 export const BillingAlert = () => {
   const { isLoggedIn } = useLoggedInUser();
   const usageQuery = useAccountUsage();
-  const meQuery = useAccount();
+  const meQuery = useAccount({ refetchInterval: 1000 });
```

Comment:
do we need to _continuously_ fetch here or is there some kind of condition where we can stop polling?

does this mean currently that every user who is on this page hits our API server once a second?

Reply:
Yeah this was a tough call, I'm not too sure if there's a better way to store state globally in our app (or even if this would help). The issue here is this billing alert is completely isolated from the billing page (we need to show it on every page) but this alert's behaviour is based on the account that gets updated via stripe webhooks (async)

Reply:
yeah, fwiw the polling seems fine as long as it stops once we have some "correct" state

Reply:
Yeah actually I saw that we can get the account on every poll so I added a start/stop condition now so we don't poll indefinitely

Thanks for the advice! 🎉